wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -125,7 +126,7 @@ library
                        Cardano.Wallet.WalletLayer.Legacy
                        Cardano.Wallet.WalletLayer.Error
                        Paths_cardano_sl_wallet_new
-  ghc-options:         -Wall
+  ghc-options:         -Wall -Werror -Wcompat -fwarn-redundant-constraints
```

Comment:
Why did you make this change?

Reply:
Because these are the settings that CI seem to be using when building. It makes sense to try and solve anything that will trigger a CI error before submitting the changes. If building with `stack`, then these GHC flags are already being used.